### PR TITLE
Add metrics collection and metrics-based dashboards info to Memcached OTel app doc

### PR DESCRIPTION
**Pull request by sumoanema**
## Purpose of this pull request

Changes to memcache-otel app doc after changes in it to add metric collection and metric-based dashboards

## Select the type of change

- [ ] Minor Changes - Typos, formatting, slight revisions
- [x] Update Content - Revisions, updating sections
- [ ] New Content - New features, sections, pages, tutorials
- [ ] Site and Tools - .clabot, version updates, maintenance, dependencies, new packages for the site (Docusaurus, Gatsby, React, etc.)

### CODE REVIEW

File: docs/integrations/databases/opentelemetry/memcached-opentelemetry.md [hunk 1]
```diff
@@ -182,17 +191,82 @@ Following is the query from Errors panel of Memcached app's overview Dashboard:
...
+   <td>Alert Condition </td>
+   <td>Recover Condition   </td>
+  </tr>
+  <tr>
+   <td>Memcached - Commands Error </td>
+   <td>This alert fires when we detect command errors.   </td>
+   <td>Critical   </td>
+   <td> &#62; 0   </td>
+   <td> &#60;&#61; 0 </td>
+  </tr>
```
Comment (kimsauce) [inline docs/integrations/databases/opentelemetry/memcached-opentelemetry.md:258]:
```suggestion
  </tr>
  <tr>
```

File: docs/integrations/databases/opentelemetry/memcached-opentelemetry.md [hunk 2]
```diff
@@ -182,17 +191,83 @@ Following is the query from Errors panel of Memcached app's overview Dashboard:
...
 ```
+## Sample metrics query
+**Total Get**
+
+```
+sumo.datasource=memcached deployment.environment=* db.cluster.name=* db.node.name=* metric=memcached.commands command=get | sum
+```
 
-## Viewing Memcached dashboards
+## Create monitors for the Sumo Logic Audit app
```
Comment (kimsauce) [inline docs/integrations/databases/opentelemetry/memcached-opentelemetry.md:201]: Shouldn't this be the Memcached app?

File: docs/integrations/databases/opentelemetry/memcached-opentelemetry.md [hunk 3]
```diff
@@ -182,17 +191,83 @@ Following is the query from Errors panel of Memcached app's overview Dashboard:
...
+
+import CreateMonitors from '../../reuse/apps/create-monitors.md';
```
Comment (kimsauce) [inline docs/integrations/databases/opentelemetry/memcached-opentelemetry.md:203]:
```suggestion
import CreateMonitors from '../../../reuse/apps/create-monitors.md';
```

File: docs/integrations/databases/opentelemetry/memcached-opentelemetry.md (hunk 2 above)
Comment (sumoanema) [reply docs/integrations/databases/opentelemetry/memcached-opentelemetry.md]: Thanks for pointing that out. Corrected.

File: docs/integrations/databases/opentelemetry/memcached-opentelemetry.md [hunk 4]
```diff
@@ -11,7 +11,7 @@ import TabItem from '@theme/TabItem';
 
 <img src={useBaseUrl('img/integrations/databases/memcached.png')} alt="Thumbnail icon" width="50"/> <img src={useBaseUrl('img/send-data/otel-color.svg')} alt="Thumbnail icon" width="45"/>
 
-The [Memcached](https://memcached.org/about) app is a logs based app that helps you monitor your Memcached clusters. Preconfigured dashboards provide insight into errors, warnings, and commands executed.
+The [Memcached](https://memcached.org/about) app is a logs based app that helps you monitor the availability, performance, health, and resource utilization of your Memcached clusters. Preconfigured dashboards provide insight into operational metrics, cache performance, resource utilization, errors, warnings, and commands executed.
```
Comment (kimsauce) [inline docs/integrations/databases/opentelemetry/memcached-opentelemetry.md:14]:
```suggestion
The [Memcached](https://memcached.org/about) app is a logs-based app that helps you monitor the availability, performance, health, and resource utilization of your Memcached clusters. Preconfigured dashboards provide insight into operational metrics, cache performance, resource utilization, errors, warnings, and commands executed.
```

File: docs/integrations/databases/opentelemetry/memcached-opentelemetry.md [hunk 5]
```diff
@@ -23,10 +23,12 @@ Memcache logs are sent to Sumo Logic through OpenTelemetry [filelog receiver](ht
 
 Following are the [Fields](/docs/manage/fields/) which will be created as part of Memcached App install if not already present.
 
-- **`db.cluster.name`**. User configured. Enter a name to identify this Memcache cluster. This cluster name will be shown in the Sumo Logic dashboards.
+- **`sumo.datasource`**. Has a fixed value of **memcached**.
 - **`db.system`**. Has a fixed value of **memcached**
 - **`deployment.environment`**. User configured. This is the deployment environment where the Memcache cluster resides. For example: dev, prod or qa.
-- **`sumo.datasource`**. Has a fixed value of **memcached**.
+- **`db.cluster.name`**. User configured. Enter a name to identify this Memcache cluster. This cluster name will be shown in the Sumo Logic dashboards.
```
Comment (kimsauce) [inline docs/integrations/databases/opentelemetry/memcached-opentelemetry.md:29]:
```suggestion
- **`db.cluster.name`**. User configured. Enter a name to identify this Memcached cluster. This cluster name will be shown in the Sumo Logic dashboards.
```

File: docs/integrations/databases/opentelemetry/memcached-opentelemetry.md [hunk 6]
```diff
@@ -84,6 +86,8 @@ In this step, you will configure the yaml file required for Memcached Collection
 
 The files are typically located in `/var/log/memcached/memcached.log`. If you're using a customized path, check the respective conf file (default location: `/etc/memcached.conf`) for this information.
 
+For metrics you are required to provide the memcached endpoint (default endpoint for memcache is localhost:11211) along with the collection_interval. Collection interval is set to 1 minute by default.
```
Comment (kimsauce) [inline docs/integrations/databases/opentelemetry/memcached-opentelemetry.md:89]:
```suggestion
For metrics, you're required to provide the Memcached endpoint (default is localhost:11211) along with the `collection_interval` (default is 1 minute).
```

File: docs/integrations/databases/opentelemetry/memcached-opentelemetry.md [hunk 7]
```diff
@@ -163,14 +167,19 @@ import LogsOutro from '../../../reuse/apps/opentelemetry/send-logs-outro.md';
 <LogsOutro/>
 
 
-## Sample log messages
+## Sample log message
```
Comment (kimsauce) [inline docs/integrations/databases/opentelemetry/memcached-opentelemetry.md:170]: These are standard headers throughout all apps docs that we keep plural by default (even if there's only one example)

```suggestion
## Sample log messages
```

File: docs/integrations/databases/opentelemetry/memcached-opentelemetry.md [hunk 8]
```diff
@@ -163,14 +167,19 @@ import LogsOutro from '../../../reuse/apps/opentelemetry/send-logs-outro.md';
...
 
 ```
 Jun 23 07:35:01 node03 memcached: \
 <31 set GFcIh47CswfCnwk3JkmJ 0 0 4096
 ```
 
-## Sample queries
+## Sample metric message
+```
```
Comment (kimsauce) [inline docs/integrations/databases/opentelemetry/memcached-opentelemetry.md:178]: These are standard headers throughout all apps docs that we keep plural by default (even if there's only one example)

```suggestion
## Sample metric messages

```
```

File: docs/integrations/databases/opentelemetry/memcached-opentelemetry.md [hunk 9]
```diff
@@ -163,14 +167,19 @@ import LogsOutro from '../../../reuse/apps/opentelemetry/send-logs-outro.md';
...
-## Sample log messages
+## Sample log message
 
 ```
 Jun 23 07:35:01 node03 memcached: \
 <31 set GFcIh47CswfCnwk3JkmJ 0 0 4096
 ```
 
-## Sample queries
+## Sample metric message
```
Comment (kimsauce) [inline docs/integrations/databases/opentelemetry/memcached-opentelemetry.md:177]: These are standard headers throughout all apps docs that we keep plural by default (even if there's only one example)

```suggestion
## Sample metric messages

```

File: docs/integrations/databases/opentelemetry/memcached-opentelemetry.md [hunk 10]
```diff
@@ -163,14 +167,19 @@ import LogsOutro from '../../../reuse/apps/opentelemetry/send-logs-outro.md';
...
+```
+{"queryId":"A","_source":"memcached-otel","_metricId":"lU4jhUa6YoiLX9eJZGHHjA","_sourceName":"Http Input","host.id":"i-0cc7753247bad73ae","host.group":"memcachedlinux","os.type":"linux","sumo.datasource":"memcached","db.system":"memcached","command":"touch","db.node.name":"ip-172-31-64-180.ec2.internal","_sourceCategory":"Labs/memcached-otel","deployment.environment":"memcachedlinux","_contentType":"Carbon2","host.name":"ip-172-31-64-180.ec2.internal","metric":"memcached.commands","_collectorId":"000000000F90B095","_sourceId":"00000000594E7C3A","unit":"{commands}","db.cluster.name":"test","_collector":"Labs - memcached-otel","max":10,"min":0,"avg":7,"sum":70,"latest":10,"count":10}
```
Comment (kimsauce) [inline docs/integrations/databases/opentelemetry/memcached-opentelemetry.md:179]:
```suggestion
{
  "queryId": "A",
  "_source": "memcached-otel",
  "_metricId": "lU4jhUa6YoiLX9eJZGHHjA",
  "_sourceName": "Http Input",
  "host.id": "i-0cc7753247bad73ae",
  "host.group": "memcachedlinux",
  "os.type": "linux",
  "sumo.datasource": "memcached",
  "db.system": "memcached",
  "command": "touch",
  "db.node.name": "ip-172-31-64-180.ec2.internal",
  "_sourceCategory": "Labs/memcached-otel",
  "deployment.environment": "memcachedlinux",
  "_contentType": "Carbon2",
  "host.name": "ip-172-31-64-180.ec2.internal",
  "metric": "memcached.commands",
  "_collectorId": "000000000F90B095",
  "_sourceId": "00000000594E7C3A",
  "unit": "{commands}",
  "db.cluster.name": "test",
  "_collector": "Labs - memcached-otel",
  "max": 10,
  "min": 0,
  "avg": 7,
  "sum": 70,
  "latest": 10,
  "count": 10
}
```

File: docs/integrations/databases/opentelemetry/memcached-opentelemetry.md [hunk 11]
```diff
@@ -182,17 +191,83 @@ Following is the query from Errors panel of Memcached app's overview Dashboard:
 | timeslice by 1h 
 | sum(ERROR) as ERROR by _timeslice
 ```
+## Sample metrics query
```
Comment (kimsauce) [inline docs/integrations/databases/opentelemetry/memcached-opentelemetry.md:194]: These are standard headers throughout all apps docs that we keep plural by default (even if there's only one example)

```suggestion
## Sample metrics queries
```

File: docs/integrations/databases/opentelemetry/memcached-opentelemetry.md [hunk 12]
```diff
@@ -182,17 +191,83 @@ Following is the query from Errors panel of Memcached app's overview Dashboard:
...
+  <tr>
+   <td>Memcached - High Memory Usage </td>
+   <td>This alert fires when the memory usage is greater than configured threshold value </td>
+   <td>Warning   </td>
+   <td> &#62;80 </td>
+   <td> &#60;&#61; 80   </td>
+  </tr>
+  <tr>
+   <td>Memcached - Cache Hit Ratio   </td>
+   <td>The hit rate is one of the most important indicators of Memcached performance. A high hit rate means faster responses to your users. If the hit rate is falling, you need quick visibility into why. This alert gets fired low cache hit ratio is less than 50%   </td>
```
Comment (kimsauce) [inline docs/integrations/databases/opentelemetry/memcached-opentelemetry.md:268]:
```suggestion
   <td>The hit rate is one of the most important indicators of Memcached performance. A high hit rate means faster responses to your users. If the hit rate is falling, you need quick visibility into why. This alert gets fired low cache hit ratio is less than 50%.</td>
```

Comment (kimsauce) [inline docs/integrations/databases/opentelemetry/memcached-opentelemetry.md:268]:
```suggestion
   <td>The hit rate is one of the most important indicators of Memcached performance. A high hit rate means faster responses to your users. If the hit rate is falling, you need quick visibility into why. This alert gets fired low cache hit ratio is less than 50%.</td>
```

File: docs/integrations/databases/opentelemetry/memcached-opentelemetry.md [hunk 13]
```diff
@@ -182,17 +191,83 @@ Following is the query from Errors panel of Memcached app's overview Dashboard:
...
+  <tr>
+   <td>Memcached - Commands Error </td>
+   <td>This alert fires when we detect command errors.   </td>
+   <td>Critical   </td>
+   <td> &#62; 0   </td>
+   <td> &#60;&#61; 0 </td>
+  </tr>
+  <tr>
+   <td>Memcached - High Memory Usage </td>
+   <td>This alert fires when the memory usage is greater than configured threshold value </td>
```
Comment (kimsauce) [inline docs/integrations/databases/opentelemetry/memcached-opentelemetry.md:261]:
```suggestion
   <td>This alert fires when the memory usage is greater than configured threshold value.</td>
```

File: docs/integrations/databases/opentelemetry/memcached-opentelemetry.md [hunk 14]
```diff
@@ -182,17 +220,88 @@ Following is the query from Errors panel of Memcached app's overview Dashboard:
...
+```
+sumo.datasource=memcached deployment.environment=* db.cluster.name=* db.node.name=* metric=memcached.commands command=get | sum
+```
+
+## Create monitors for the Sumo Logic Memcached app
 
-## Viewing Memcached dashboards
+import CreateMonitors from '../../../reuse/apps/create-monitors.md';
+
+## Viewing Memcached Dashboards
```
Comment (JV0812) [inline docs/integrations/databases/opentelemetry/memcached-opentelemetry.md:234]:
```suggestion
## Viewing Memcached dashboards
```

File: docs/integrations/databases/opentelemetry/memcached-opentelemetry.md [hunk 15]
```diff
@@ -182,17 +220,88 @@ Following is the query from Errors panel of Memcached app's overview Dashboard:
...
 ### Overview
 
-The **Memcached - Overview** dashboard provides an at-a-glance view of the Memcached errors, client protocol, and command executed.
+The **Memcached - Overview** dashboard provides an at-a-glance view of the Memcached server status, error logs, and database metrics.
+
+<img src='https://sumologic-app-data-v2.s3.amazonaws.com/dashboards/Memcached-OpenTelemetry/Memcached-Overview.png' alt="Memcached dashboards" />
+
+### Operations
+
+The **Memcached - Operations** Dashboard provides detailed analysis on connections, thread requested, network bytes, table size.
```
Comment (JV0812) [inline docs/integrations/databases/opentelemetry/memcached-opentelemetry.md:244]:
```suggestion
The **Memcached - Operations** dashboard provides detailed analysis on connections, thread requested, network bytes, and table size.
```

File: docs/integrations/databases/opentelemetry/memcached-opentelemetry.md [hunk 16]
```diff
@@ -182,17 +220,88 @@ Following is the query from Errors panel of Memcached app's overview Dashboard:
...
+  <tr>
+   <td>Memcached - Cache Hit Ratio   </td>
+   <td>The hit rate is one of the most important indicators of Memcached performance. A high hit rate means faster responses to your users. If the hit rate is falling, you need quick visibility into why. This alert gets fired low cache hit ratio is less than 50%.</td>
+   <td>Critical </td>
+   <td> &#60;&#61;50   </td>
+   <td> &#62;50   </td>
+  </tr>
+  <tr>
+   <td>Memcached - High Number of Connections   </td>
+   <td>This alert fires when the number of current connection for memcached exceed given threshold. </td>
```
Comment (JV0812) [inline docs/integrations/databases/opentelemetry/memcached-opentelemetry.md:302]:
```suggestion
   <td>This alert fires when the number of current connection for Memcached exceed the given threshold. </td>
```

File: docs/integrations/databases/opentelemetry/memcached-opentelemetry.md [hunk 17]
```diff
@@ -182,17 +220,88 @@ Following is the query from Errors panel of Memcached app's overview Dashboard:
...
+  <tr>
+   <td>Memcached - High Memory Usage </td>
+   <td>This alert fires when the memory usage is greater than configured threshold value.</td>
+   <td>Warning   </td>
+   <td> &#62;80 </td>
+   <td> &#60;&#61; 80   </td>
+  </tr>
+  <tr>
+   <td>Memcached - Cache Hit Ratio   </td>
+   <td>The hit rate is one of the most important indicators of Memcached performance. A high hit rate means faster responses to your users. If the hit rate is falling, you need quick visibility into why. This alert gets fired low cache hit ratio is less than 50%.</td>
```
Comment (JV0812) [inline docs/integrations/databases/opentelemetry/memcached-opentelemetry.md:295]:
```suggestion
   <td>The hit rate is one of the most important indicators of Memcached performance. A high hit rate means faster responses to your users. If the hit rate is falling, you need quick visibility into why. This alert gets fired if the low cache hit ratio is less than 50%.</td>
```

File: docs/integrations/databases/opentelemetry/memcached-opentelemetry.md [hunk 18]
```diff
@@ -182,17 +220,88 @@ Following is the query from Errors panel of Memcached app's overview Dashboard:
...
+  <tr>
+   <td>Memcached - Commands Error </td>
+   <td>This alert fires when we detect command errors.   </td>
+   <td>Critical   </td>
+   <td> &#62; 0   </td>
+   <td> &#60;&#61; 0 </td>
+  </tr>
+  <tr>
+   <td>Memcached - High Memory Usage </td>
+   <td>This alert fires when the memory usage is greater than configured threshold value.</td>
```
Comment (JV0812) [inline docs/integrations/databases/opentelemetry/memcached-opentelemetry.md:288]:
```suggestion
   <td>This alert fires when the memory usage is greater than the configured threshold value.</td>
```

**Review by JV0812 — CHANGES_REQUESTED** [review]
Minor fixes

**Review by JV0812 — APPROVED** [review]
LGTM

**Review by jpipkin1 — APPROVED** [review]

**Pull request merged**